modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/user/mgt/UserMgtTestCase.java

```diff
@@ -42,6 +42,8 @@ public class UserMgtTestCase extends ISIntegrationTest {
 	private UserManagementClient userMgtClient;
 	private AuthenticatorClient logManger;
 
+	//Claim Uris
```

Comment:
This comment seems unnecessary since the variable name is self-descriptive. We can remove or let's fix the formatting in the comment if we are keeping it. Check the same in the other classes as well.

Reply:
Removed this and similar comments since it seems unnecessary to add as a comment